src/java.base/share/classes/sun/security/util/SignatureUtil.java

```diff
@@ -47,32 +47,39 @@
 public class SignatureUtil {
 
     /**
-     * Convert OID.1.2.3.4 or 1.2.3.4 to its matching stdName.
+     * Convert OID.1.2.3.4 or 1.2.3.4 to its matching stdName, and literal
+     * name to upper case.
```

Comment:
It sounds like you only convert literal name to upper case and return stdName for OIDs.

Reply:
Hm, it makes sense to me.  Updated in the next commit.